### PR TITLE
manual.grid.tf updated links

### PR DESCRIPTION
**Pull request by Mik-TF**
# Work Done

- Updated the links for the new manual directory tree.

# Moment of Merge

As discussed with @xmonader, we need to merge this for Monday release, at the same time that we merge the info_grid PR [here](https://github.com/threefoldtech/info_grid/pull/444) and the www_threefold_io PR [here](https://github.com/threefoldfoundation/www_threefold_io/pull/367)

### CODE REVIEW

File: packages/playground/public/info/minting.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ The TFT minting address on Stellar Chain:
 
 <br />
 
-For more questions about minting, please contact the [support team](https://threefold.io/support/) or click on the chat icon.
+For more questions about minting, please contact the [support team](https://threefoldfaq.crisp.help/en/) or click on the chat icon.
```
Comment (amiraabouhadid) [inline packages/playground/public/info/minting.md:20]: @AhmedHanafy725 do we use crisp here or update it to another link?

Comment (AhmedHanafy725) [reply packages/playground/public/info/minting.md:20]: please use threefold.io instead

Comment (xmonader) [reply packages/playground/public/info/minting.md:20]: unblock it with the crisp url, the marketing team will be restoring a support page soonish

**Comment by amiraabouhadid** [comment]
Please refer to #2397 for the updated manual links but from the development base branch. Please note that all .crisp links have been updated to threefold.io

**Comment by Mik-TF** [comment]
Great. Closing this PR and delete the branch then. Thanks.

**Pull request closed without merging**